test/conftest.py

```diff
@@ -69,7 +69,9 @@ def expect_nav_items() -> List[Dict[str,str]]:
         {'title': 'Boost!', 'href': '/boost'},
         {'title': 'Garage Sale', 'href': 'https://www.jupitergarage.com/'},
         {'title': 'Membership', 'href': '/membership'},
-        {'title': 'Archive', 'href': '/archive'},
```

Comment:
nice catch. We still need to put the E2E in our mental image of all PRs and features and changes.....

Reference: addresses failed E2E based on change in #399